doc/source/ray-core/examples/batch_training.ipynb

```diff
@@ -397,7 +382,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 10,
```

Comment:
For the text in the section above, can you introduce that this is the part where we start stitching all the code together to actually run the job?